const deleteUser = () => {
    try {

    } catch (error) {

    }
}

module.exports = deleteUser